Feature/number guessing game

mercurrie opened the pull request:
completed feature, number_guessing_game, utilizing flask

mercurrie committed:
update

mercurrie committed:
basic number guessing game

program utilizes flask to perform the functions of a basic number guessing game

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,30 @@
+
+@app.route("/<int:guess>")
+def guess_number(guess):
+    if guess > random_number:
```
Looks good.  Some of this is duplicated so I'd probably create a dictionary with values for: `message`, `color`, and `image`, stash it at the top, and then grab those values during the game.

mercurrie replied:
I included a json file to hold the data for the styling, img elements, and message outputs

michaelhousman reviewed (commented):
Pretty simple game but this looks good.

mercurrie committed:
refactor

included json file to hold styling and img data.
utilized json data in main.py to reduce repetition.

mercurrie committed:
refactor

included json data for messages
